src/js/components/landing/blocks/LandingRequirementsBody.jsx

```diff
@@ -86,9 +86,9 @@ export default class LandingRequirementsBody extends React.Component {
 						<li>Error Codes and Messages</li>
 						<li><a href={resources} target="_blank" rel="noopener noreferrer">DATA Act Information Model Schema (DAIMS)</a> resources related to FABS. See:
 							<ul>
-								<li><a href={practices} target="_blank" rel="noopener noreferrer">DAIMS Practices &amp; Procedures</a></li>
```

Comment:
Since you remove the `practices` variable, you should delete where it's being declared because I don't think it's used anywhere.